jwthenticator/tokens.py

```diff
@@ -12,7 +12,7 @@
 from jwthenticator.schemas import JWTPayloadData, RefreshTokenData
 from jwthenticator.exceptions import InvalidTokenError, MissingJWTError
 from jwthenticator.consts import JWT_ALGORITHM, REFRESH_TOKEN_EXPIRY, JWT_LEASE_TIME, JWT_AUDIENCE, DB_URI
-
+from jwthenticator.loop_management import db_lock
```

Comment:
Is this needed? its not mentioned in the rest of the file

Reply:
no, i will remove it